diff --git a/.zkp_devnet/meta_data.json b/.zkp_devnet/meta_data.json
index 9be15be53..bab3a5707 100644
--- a/.zkp_devnet/meta_data.json
+++ b/.zkp_devnet/meta_data.json
@@ -1,5 +1,5 @@
 {
   "genesisHash": "bd79d1e2dd97e7ca0940fcd719f52baa473859661b5d7d4f092603d1d3fc67ad",
   "vksCommitment": "0e602abf5b642b77f66a4cc7f452c9339b61f68bac7880c8258f4f808130c8a3ee5ad8a8a82703855bc94cc5b36a7df79882020617327a60e8905a41b86a269e7b680e23f4c1b30536f2fe27f6fd45110a557f3b42f73aba064ba46a3a85807f8271411a56da6edfac40941c7a8e53645154c0678391a9634133166410330ba6fe10e4a1e8157de3e155d1bfeb909343c91f5931d46c383a09b072de91f10bb3a78690afb648ef9086bb1f1c18443b6d2d699f1f110f5acaf37dc249cb80",
-  "blocksPerEpoch": 43200
+  "blocksPerEpoch": 480
 }
\ No newline at end of file
diff --git a/genesis/src/genesis/dev-albatross-4-validators.toml b/genesis/src/genesis/dev-albatross-4-validators.toml
index fbc3b571d..2604e840f 100644
--- a/genesis/src/genesis/dev-albatross-4-validators.toml
+++ b/genesis/src/genesis/dev-albatross-4-validators.toml
@@ -89,6 +89,36 @@ staker_address = "NQ64 VP65 7809 JC9D X4XP QDRU KYTN 79JN 46A5"
 balance = 1_000_00000
 delegation = "NQ83 BPY4 JPJH 5B1Y NYKQ 32V1 84L4 YU35 1VTL"
 
+[[stakers]]
+staker_address = "NQ69 F067 AXJ7 TJTE 3J80 5GCY 6CMX 1FEK 6MCR"
+balance = 100000000
+delegation = "NQ20 TSB0 DFSM UH9C 15GQ GAGJ TTE4 D3MA 859E"
+inactive_balance = 0
+
+[[stakers]]
+staker_address = "NQ49 EASY CBKX 3LYM 1YY2 DSL7 L8XR DHBJ NGVL"
+balance = 111369400000
+delegation = "NQ20 TSB0 DFSM UH9C 15GQ GAGJ TTE4 D3MA 859E"
+inactive_balance = 0
+
+[[stakers]]
+staker_address = "NQ38 8S48 6YSG CUED L6D6 F16T 7NHR 1C42 7XAX"
+balance = 1000000000
+delegation = "NQ20 TSB0 DFSM UH9C 15GQ GAGJ TTE4 D3MA 859E"
+inactive_balance = 0
+
+[[stakers]]
+staker_address = "NQ79 3E8M NKXN 8YUD R9C2 EY4E 23N9 4L5K 62QS"
+balance = 1178400000
+delegation = "NQ20 TSB0 DFSM UH9C 15GQ GAGJ TTE4 D3MA 859E"
+inactive_balance = 0
+
+[[stakers]]
+staker_address = "NQ48 XNXR J3D5 CVYG 0UPC PTRX AP56 V16T PTA1"
+balance = 995200000
+delegation = "NQ20 TSB0 DFSM UH9C 15GQ GAGJ TTE4 D3MA 859E"
+inactive_balance = 0
+
 [[basic_accounts]]
 address = "NQ55 X103 EJXG 3S2U 3JNE L779 560T 3LNE 7S60"
 # private_key = "a24591648e20642fe5107d0285c1cc35d67e2033a92566f1217fbd3a14e07abc"
diff --git a/primitives/src/policy.rs b/primitives/src/policy.rs
index c9c2dd24e..dc83485f2 100644
--- a/primitives/src/policy.rs
+++ b/primitives/src/policy.rs
@@ -645,9 +645,9 @@ impl Default for Policy {
     fn default() -> Self {
         Policy {
             blocks_per_batch: 60,
-            batches_per_epoch: 720,
+            batches_per_epoch: 8,
             state_chunks_max_size: 1000,
-            transaction_validity_window: 120,
+            transaction_validity_window: 2,
             genesis_block_number: 0,
         }
     }
